Controller.php

```diff
@@ -346,6 +350,54 @@ public function exportContainerVersion()
         return $result;
     }
 
+    public function copyContainerDialog()
+    {
+        $this->checkSitePermission();
+        $this->accessValidator->checkWriteCapability($this->idSite);
```

Comment:
@snake14 We are checking write capability here, but in vue we are checking only if its superUser, is this intended ?

Reply:
Good catch! I forgot to adjust this check when we decided to change this feature to only be for superusers. I just fixed it :+1: